src/main/java/org/fundacionjala/trello/pages/board/BoardPage.java

```diff
@@ -17,6 +19,7 @@
     private static final String BOARD_HEADER = "div.board-header";
     private static final String BOARD_CANVAS = "#board";
     private static final String ADD_LIST = "div.js-add-list span.icon-add";
+    private static final String GET_LIST = "//textarea[contains(text(), '%s')]/parent::div";
```

Comment:
I believe taking a few more parameters is better for locating an element. like the class of the web element or some ancestor.
If the locator is too general, it could match an unexpected web element.
In this case, you can also match this locator with the item "Card", when you click the Edit button on the card.

Reply:
done